cEP-0005.md

```diff
@@ -102,18 +102,25 @@ class RedundancyBear(LocalBear):
     # This bear is supposed to be able to fix unused imports...
     FIX_ASPECTS = {Root.Redundancy.UnusedImport}
     # ... and detect code clones.
-    DETECT_ASPECTS = {Root.Redundancy.CloneRedundancy}
+    DETECT_ASPECTS = {Root.Redundancy.Clone}
 
     # Aspects are passed as parameter
     def run(self, filename, file, aspects):
```

Comment:
see below, I like the idea, that would be an own cEP IMO and can be implemented orthogonally to this, so we can do it later.